Debian-local:
  Use the Development section, for consistency with other Debian packages.

Index: gdb-6.3/gdb/doc/gdb.texinfo
===================================================================
--- gdb-6.3.orig/gdb/doc/gdb.texinfo	2004-10-08 15:19:03.000000000 -0400
+++ gdb-6.3/gdb/doc/gdb.texinfo	2004-11-10 00:38:00.343924014 -0500
@@ -38,7 +38,7 @@
 
 @c This is a dir.info fragment to support semi-automated addition of
 @c manuals to an info tree.
-@dircategory Software development
+@dircategory Development
 @direntry
 * Gdb: (gdb).                     The GNU debugger.
 @end direntry
Index: gdb-6.3/gdb/doc/gdbint.texinfo
===================================================================
--- gdb-6.3.orig/gdb/doc/gdbint.texinfo	2004-10-12 15:14:31.000000000 -0400
+++ gdb-6.3/gdb/doc/gdbint.texinfo	2004-11-10 00:37:59.165154226 -0500
@@ -1,7 +1,7 @@
 \input texinfo   @c -*- texinfo -*-
 @setfilename gdbint.info
 @include gdb-cfg.texi
-@dircategory Software development
+@dircategory Development
 @direntry
 * Gdb-Internals: (gdbint).	The GNU debugger's internals.
 @end direntry
Index: gdb-6.3/gdb/doc/stabs.texinfo
===================================================================
--- gdb-6.3.orig/gdb/doc/stabs.texinfo	2004-09-21 17:06:37.000000000 -0400
+++ gdb-6.3/gdb/doc/stabs.texinfo	2004-11-10 00:37:59.874015787 -0500
@@ -5,7 +5,7 @@
 
 @c This is a dir.info fragment to support semi-automated addition of
 @c manuals to an info tree.
-@dircategory Software development
+@dircategory Development
 @direntry
 * Stabs: (stabs).                 The "stabs" debugging information format.   
 @end direntry
